pkg/errors/errors.go

```diff
@@ -84,14 +84,12 @@ func HandleExitCoder(err error) {
 	}
 
 	if exitErr, ok := err.(ExitCoder); ok {
-		if err.Error() != "" {
-			fmt.Fprintln(ErrWriter, err)
+		if errMsg := err.Error(); errMsg != "" {
+			fmt.Fprintln(ErrWriter, errMsg)
 		}
 		OsExiter(exitErr.ExitCode())
-		return
+	} else {
```

Comment:
you can remove `else` here? WDYT?

Reply:
i think you are right, let me try updating it and testing again